sdk/storage/ci.yml

```diff
@@ -12,6 +12,8 @@ trigger:
     - sdk/storage/Azure.Storage.DataMovement/
     - sdk/storage/Azure.Storage.DataMovement.Blobs/
     - sdk/storage/Azure.Storage.DataMovement.Files/
+    exclude:
+    - sdk/storage/Azure.ResourceManager.Storage/
```

Comment:
This change should be reverted.   You shouldn't be altering files outside of your /sdk area.